scripts/copy_druid_docs.py

```diff
@@ -90,15 +85,14 @@ def is_it_latest(druid_version, source_directory, destination_directory_latest):
 
     if is_latest == 'y':
         print("Also copying the docs to docs/latest.")
-        copy_tree(source_directory+'/docs', destination_directory_latest)
+        subprocess.run(["rsync", "-av", "--delete", f"{source_directory}/docs/", destination_directory_latest])
```

Comment:
I'm not sure if we actually want the list of files being sent? I'm guessing no. If so, I can remove `-av` from this and 106